### PR TITLE
LPS-141235 Add test that verifies remote app type IFrame can display property fields

### PR DESCRIPTION
**Pull request by sergiojimcos**
**Test Name:**
RemoteApps#IframeTypeCanDisplayProperFields

**Description:**
Remote App type IFrame can display property fields.

**Test scenario:**
Given: **Remote Apps** Add page.
When: Type is **IFrame**.
Then: **HTML Element Name** and **CSS URL** fields are not present.
When: select type **Custom Element**.
Then: **HTML Element Name** and **CSS URL** fields are present.

**JIRA Ticket:**
https://issues.liferay.com/browse/LPS-141235

### CODE REVIEW

**Comment by sergiojimcos** [comment]
ci:test:frontend:nocompile

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase [hunk 1]
```diff
@@ -79,4 +79,28 @@ definition {
...
+	test IframeTypeCanDisplayProperFields {
+		property portal.acceptance = "true";
+
+		var customElementType = "Custom Element";
+
+		RemoteApps.goToRemoteAppsPortlet();
+
+		LexiconEntry.gotoAdd();
+
+		AssertNotVisible(locator1 = "RemoteAppsEntry#HTML_ELEMENT_INPUT");
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase:93]: To make the test more robust, you'll probably want to verify that Iframe type is selected and if not then select Iframe type before asserting not visible.

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/remoteapps/RemoteAppsEntry.path [hunk 2]
```diff
@@ -20,6 +20,16 @@
...
+<tr>
+	<td>HTML_ELEMENT_INPUT</td>
+	<td>//input[contains(@id,'customElementHTMLElementName')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>CSS_ELEMENT_INPUT</td>
+	<td>//input[contains(@id,'customElementCSSURLs')]</td>
+	<td></td>
+</tr>
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/remoteapps/RemoteAppsEntry.path:32]: This is usually good xpath, but my preference is to use some common xpaths found in liferay-portal/portal-web/test/functional/com/liferay/portalweb/paths/pathlib such as TextInput#ANY. I'd prefer we don't create more xpaths if we have a common pattern that can fit in well.

Please see as example: https://github.com/liferay/liferay-portal/blob/7eac9cd5daa98add4511fc853a91cd41e8f6eec6/portal-web/test/functional/com/liferay/portalweb/macros/RemoteAppsEntry.macro#L4-L18

**Review by john-co — CHANGES_REQUESTED** [review]
Looks good overall! Please review the comments and make the requested changes.

**Comment by sergiojimcos** [comment]
ci:test:frontend:nocompile

**Review by john-co — APPROVED** [review]
Looks good to me! :)

**Comment by john-co** [comment]
ci:forward

**Pull request closed without merging**